contracts/cw20-ics20/src/contract.rs

```diff
@@ -103,7 +122,7 @@ pub fn execute_transfer(
     );
     packet.validate()?;
 
-    // prepare message
+    // prepare message and set proper gas limit
```

Comment:
I see the comment about setting gas, but I don't see single line of code changed. Is it just encapsulated in the `packet`? Kind of guess, but I want to understand the comment change.

Reply:
you are right. I need to remove it.

I started adding gas limit here, then realized I need to add it when seeing the cw20 token (2 places in ibc.rs) not when making the ibc packet. It should be in ibc_packet_receive and in ibc_packet_failure

Reply:
Ok, then just remove commend and it is fine. I accept the PR then as this is not a functional thing.

Reply:
Ok, then just remove commend and it is fine. I accept the PR then as this is not a functional thing.